R/pkg/_pkgdown.yml

```diff
@@ -0,0 +1,293 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+url: ~
+
+template:
+  package: preferably
```

Comment:
Demo version uses Bootstrap 3, because `preferably` doesn't fully support `pkgdown>1.6` and doesn't work with Bootstrap 5, but such upgrade is an option.

Reply:
I noticed that this issue is linked to my repo, so I thought I would come here and say that I'm working on an update to support BS5. In the meantime, I just pushed a minor patch that fixes some minor issues from upgrading to `{pkgdown} 2.0`! :-)

Reply:
Thanks @amirmasoudabdol!

Reply:
{preferably} now supports {pkgdown} 2.0, and Bootstrap 5! 🎉

It is not yet on CRAN though, I'm waiting for their approval, etc.

Reply:
Thanks for the update @amirmasoudabdol!